srcpkgs/bc/template

```diff
@@ -1,17 +1,20 @@
 # Template file for 'bc'
 pkgname=bc
 version=1.07.1
-revision=3
+revision=4
 build_style=gnu-configure
 configure_args="--with-readline"
 hostmakedepends="ed flex"
 makedepends="readline-devel"
-short_desc="An arbitrary precision numeric processing language"
+short_desc="Arbitrary precision numeric processing language"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
+license="GPL-3.0-or-later"
 homepage="http://www.gnu.org/software/${pkgname}/"
-license="GPL-3"
 distfiles="${GNU_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
 checksum=62adfca89b0a1c0164c2cdca59ca210c1d44c3ffc46daf9931cf4942664cb02a
+alternatives="
+ bc:bc:/usr/bin/bc
```

Comment:
you created a bc alternatives that symlinks bc to... bc

Reply:
I guess I did not understand the docs. Oops.